### ISSUE TITLE
Select favorites are unable to be deselected by keyboard

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#6927]
**Describe the issue. What is the expected and unexpected behavior?**
The favorites' stars are unable to be deselected by keyboard. You can select them by pressing Enter, but there seems no way to deselect them once they are selected.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Go to [Favorites example](https://www.patternfly.org/v4/components/select#favorites), select a star by keyboard and then try to deselect it. 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
